src/joyride/nrepl.cljs

```diff
@@ -67,21 +67,22 @@
             "status" ["done"]}))
 
 (def pretty-print-fns-map
-  {"clojure.core/prn" prn
-   "clojure.pprint/pprint" pp/pprint
-   "cljs.pprint/pprint" pp/pprint
-   "cider.nrepl.pprint/pprint" pp/pprint})
+  {"clojure.core/pr" pr
+   "clojure.core/prn" prn
+   "clojure.pprint/pprint" pp/write
+   "cljs.pprint/pprint" pp/write
```

Comment:
I think none of the above values will be ever used, so let's remove them
Only the below is sufficent and pp/write seems like the good thing to do there